### DISCUSSION TITLE
RME Plugin does'nt show in Steamdeck-list

### DISCUSSION THREAD

**Discussion by MrKeys14** [shells-dw/streamdeck-totalmix#37] [Q&A]
Hello

I'm trying to get the plugin to work in Steamdeck, but after installing it does not show in the list somehow,
I have got the latest version of Steamdeck.

**Comment by Roman01215** [comment]
hi have same problem

**Comment by shells-dw** [comment]
Hm, can't really reproduce on latest Windows 11 with latest Stream Deck software. Works fine for me.

Are you  guys using the App Store version or the plugin from [here](https://github.com/shells-dw/streamdeck-totalmix/releases)?
They should be the same version anyhow, but it might help to narrow it down knowing which version doesn't work for you. You could also try to install the other version and see if that fixes it (meaning if the App Store version doesn't work, try installing from the GitHub link above and vice versa).

If they both don't work, you could try to close Stream Deck software and delete the plugin folder (de.shells.totalmix.sdPlugin) manually from %appdata%\Elgato\StreamDeck\Plugins and install it again. Very very rarely I had a similar issue during debugging (with way older versions, long ago though) that something remains in the plugin folder and thus the Stream Deck client thinks it would be installed and doesn't try to properly install it - deleting the folder manually would mitigate that issue.

Anyhow, if nothing works, report back with more details, I'm thinking about stuff like:
- Windows version
- is the plugin folder created when installing despite the client not showing it
- if it is, should be a "pluginlog.log" file in the plugin folder, if it is, look in there and see if it says something interesting (and/or attach it here)
- if that doesn't provide any info, in the Stream Deck folder on the level of "plugins" there is a "log" folder that contains the Stream Deck log (StreamDeck0.log should be the most recent one) - look in there and see if it states what the issue might be (and/or attach that file here)
- (added after posting): also, just in case, what's your system language? I don't really think that makes a difference, but who knows, maybe you both use the same system language and that's causing the issue somehow. I know it works on (any) English, Japanese and German localizations, but obviously I couldn't test all languages. Just an idea I had. Strange that 2 people report the same issue at the same time despite it appears to be still working for me 🤷

**Comment by MrKeys14** [comment]
Hi Dominik,

Somehow a windows restart did the job.
I can see them now in Streamdeck.

Thank you for your efforts,

Paul



Verzonden vanuit Mail<https://go.microsoft.com/fwlink/?LinkId=550986> voor Windows

Van: Dominik ***@***.***>
Verzonden: vrijdag 18 augustus 2023 09:00
Aan: ***@***.***>
CC: ***@***.***>; ***@***.***>
Onderwerp: Re: [shells-dw/streamdeck-totalmix] RME Plugin does'nt show in Steamdeck-list (Discussion #37)


Hm, can't really reproduce on latest Windows 11 with latest Stream Deck software. Works fine for me.

Are you guys using the App Store version or the plugin from here<https://github.com/shells-dw/streamdeck-totalmix/releases>?
They should be the same version anyhow, but it might help to narrow it down knowing which version doesn't work for you. You could also try to install the other version and see if that fixes it (meaning if the App Store version doesn't work, try installing from the GitHub link above and vice versa).

If they both don't work, you could try to close Stream Deck software and delete the plugin folder (de.shells.totalmix.sdPlugin) manually from %appdata%\Elgato\StreamDeck\Plugins and install it again. Very very rarely I had a similar issue during debugging (with way older versions, long ago though) that something remains in the plugin folder and thus the Stream Deck client thinks it would be installed and doesn't try to properly install it - deleting the folder manually would mitigate that issue.

Anyhow, if nothing works, report back with more details, I'm thinking about stuff like:

  *   Windows version
  *   is the plugin folder created when installing despite the client not showing it
  *   if it is, should be a "pluginlog.log" file in the plugin folder, if it is, look in there and see if it says something interesting (and/or attach it here)
  *   if that doesn't provide any info, in the Stream Deck folder on the level of "plugins" there is a "log" folder that contains the Stream Deck log (StreamDeck0.log should be the most recent one) - look in there and see if it states what the issue might be (and/or attach that file here)

**Comment by shells-dw** [comment]
> Hi Dominik, Somehow a windows restart did the job. I can see them now in Streamdeck. Thank you for your efforts, Paul 

Hey Paul,
love to hear it! Thanks for reporting back. Hope the plugin helps improve your workflow :)

Cheers buddy
